test/serverTest.js

```diff
@@ -158,25 +158,111 @@ describe('app',  () =>  {
 });
 
 describe('generate', function() {
-  describe('.createDistDir()', function() {
-    this.timeout(200000);
+  describe('.create different event sites and copy assets of overview site', function() {
+    this.timeout(800000);
```

Comment:
Earlier, we were generating a single event and chose 2 minutes as the timeout limit. Now, we are generating 4 events so I thought it would be good if we give 8 minutes as the timeout limit. Practically, it doesn't take that much time for it to complete. This is just for when the Travis servers are working really slow and not downloading the assets fast enough. If it completes before this time, it will continue to the next test/finish so it should be fine. Thanks!!